x-pack/plugins/index_management/__jest__/components/index_table.test.js

```diff
@@ -16,7 +16,7 @@ import { findTestSubject } from '@elastic/eui/lib/test';
       Could not load worker ReferenceError: Worker is not defined
           at createWorker (/<path-to-repo>/node_modules/brace/index.js:17992:5)
  */
```

Comment:
I ran the tests locally and it doesn't seem that this comment is relevant anymore, so we can get rid of it as well